corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditorMain.java

```diff
@@ -25,6 +25,7 @@ private enum OperationType {
         infoTable,
         showTable,
         listenOnTable,
+        dropTable,
```

Comment:
the wrapper script currently uses 'dropTable' for the clear operation.  Using the same name here can be confusing.  Can we rename this or the clearTable operation?

Reply:
maybe i am missing something but my understanding is the wrapper script just takes an operation which it directly passes on to the java main method, isn't that right?

Reply:
Also I did not remove clearTable - it is still there right beside dropTable

Reply:
sorry for the confusion.. I thought that the current operation to delete all entries is called 'dropTable' in the script.  But it is called 'clearTable.